object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionAnnotatedTypeFactory.java

```diff
@@ -311,30 +307,14 @@ public Void visitNewClass(NewClassTree tree, AnnotatedTypeMirror type) {
 
       // we override this method to handle a constructor call inside a generated toBuilder
```

Comment:
Can you change this comment to make it more generic, since the code here now doesn't have anything to do with AutoValue?